core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -511,6 +516,8 @@ object ConfigCommand extends Config {
       .ofType(classOf[String])
     val alterOpt = parser.accepts("alter", "Alter the configuration for the entity.")
     val describeOpt = parser.accepts("describe", "List configs for the given entity.")
+    val allOpt = parser.accepts("all", "List all configs (static and dynamic) for the given entity")
```

Comment:
One thing to note is that there's two paths for handling configs, one being the original direct-to-ZK, and the newer one using the admin client to handle the updates. The ZK path doesn't have an easy way for gathering the layered configs, which means implementing --all in the ZK path will be non-trivial. You're welcome to attempt this, but perhaps instead disable the flag for the ZK usage :).

Reply:
Good catch! I've updated PR so we explicitly check for `bootstrap-server`